content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
+accept **online** and **point-of-sale** payments.
```

Comment:
that offers several online payment solutions/methods/whatever -> don't mention pos